sdk/servicebus/service-bus/test/stress/.helmignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Not familiar with this - does .helmignore complement .gitignore?

If not, then we probably also want to avoid accidentally packaging up .env files, for instance.

Reply:
Added it to be safe. Normally helmignore is just for `helm package`, which we don't do in this context. However, running `helm upgrade|install|template` will do a recursive walk, which creates annoying messages when there are symlinks in the `node_modules` directory:

```
walk.go:74: found symbolic link in path: /home/ben/sdk/azure-sdk-for-js/sdk/servicebus/service-bus/test/stress/node_modules/.bin/semver resolves to /home/
ben/sdk/azure-sdk-for-js/sdk/servicebus/service-bus/test/stress/node_modules/semver/bin/semver
```

Reply:
I updated the helm ignore to just ignore `app/` since I moved things in there related to your other comment (given the manifests only reference the docker file, which contains the files).